tests/end-to-end/external-solution-integration/pkg/testkit/testservice.go

```diff
@@ -103,6 +105,67 @@ func (ts *TestService) checkValue() (int, error) {
 	return response.Counter, nil
 }
 
+func (ts *TestService) CheckAllReceivedEvents() error {
```

Comment:
Does it make sense to rename the function to something like `DumpAllReceivedEvents`?

Reply:
Yes, renamed the function and added documentation for clarity